grid/src/List/ListItem.js

```diff
@@ -33,7 +33,37 @@ const ListItem = ({
     rowSelector,
     rowActions,
     expandableColumn
-}) => {
+}: {
+    row: Object,
+    idAttribute: String,
+    style: Object,
+    theme: string,

```

Comment:
Capital S is needed for String - I guess. Please check line 38.

Reply:
Yes thats correct and I have changed into Capital S